python_ta/reporters/templates/template.txt

```diff
@@ -35,12 +35,17 @@
 			</div>
 			<u1>
 				{% for message in file.code %}
+				    <div class="container">
+				    <div class="slider">
```

Comment:
indent nested elements, and vertically match the opening tags with their closing tags.